### DISCUSSION TITLE
If immediate is on, does the animation loop still run every animation frame?

### DISCUSSION THREAD

**Discussion by oveddan** [pmndrs/react-spring#1697] [Support]
I have an app where I may have around 100 components using useSpring.  I want to be able to turn animations on/off to help with performance.

If I set `immediate` to true, does the animation loop still run when calling `useSpring` what about when an animation reaches the to (finishes)?

**Comment by joshuaellis** [comment]
I'm not sure I understand your question, but if you don't want to run animations, you can set the global `skipAnimation` to be `true`:

```js
import { Globals } from 'react-spring'

Globals.assign({
    skipAnimation: true,
})
```
